geode-lucene/src/main/java/org/apache/geode/cache/lucene/internal/LuceneIndexFactoryImpl.java

```diff
@@ -62,4 +63,7 @@ public void create(final String indexName, final String regionPath) {
     service.createIndex(indexName, regionPath, fields);
 
   }
+
+  @Override
+  public void setLuceneSerializer(LuceneSerializer luceneSerializer) {}
```

Comment:
I guess you meant to not do anything with the LuceneSerializer here yet? Do you want to "return this;" for now?

Reply:
I will change.